fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportPartitionMapper.java

```diff
@@ -269,6 +276,13 @@ public PartitionPlan mapPartitions() throws Exception {
             Properties p = new Properties();
             p.setProperty(Constants.IMPORT_PARTITTION_WORKITEM, fhirDataSource.getUrl());
             p.setProperty(Constants.IMPORT_PARTITTION_RESOURCE_TYPE, fhirDataSource.getType());
+            p.setProperty(Constants.COS_API_KEY, cosApiKeyProperty);
+            p.setProperty(Constants.COS_SRVINST_ID, cosSrvinstId);
+            p.setProperty(Constants.COS_ENDPOINT_URL, cosEndpointUrl);
+            p.setProperty(Constants.COS_LOCATION, cosLocation);
+            p.setProperty(Constants.COS_OPERATIONOUTCOMES_BUCKET_NAME, cosOperationOutcomesBucketName);
+            p.setProperty(Constants.COS_IS_IBM_CREDENTIAL, cosCredentialIbm);
+
```

Comment:
Mapper already got these properties from injection, don't need to pass through partition properties.

Reply:
That's not what I found, I found the properties were empty when it went to the step execution

Reply:
backed out this change.

Reply:
I don't really think this is correct, I mean if so, the import job should have always failed.

Reply:
sorry, I mean the chunkjob steps already got these properties from injection. don't need to be set by the mapper.  otherwise, the import job should have never succeeded from the beginning ...

Reply:
Yes, no worries, it was confused with the Collector problem. All fixed now